completions/ssh-keygen

```diff
@@ -60,17 +60,39 @@ _comp_cmd_ssh_keygen()
             ;;
         -*O)
             if [[ $cur != *=* ]]; then
-                _comp_compgen -- -W 'clear critical: extension: force-command=
-                    no-agent-forwarding no-port-forwarding no-pty no-user-rc
-                    no-x11-forwarding permit-agent-forwarding
-                    permit-port-forwarding permit-pty permit-user-rc
-                    permit-X11-forwarding no-touch-required source-address=
-                    verify-required
-
-                    lines= start-line= checkpoint= memory= start= generator=
-
-                    application= challenge= device= no-touch-required resident
-                    user= write-attestation='
+                local -a opts=()
+                case ${words[*]} in
+                    *\ -*M\ *)
```

Comment:
I think we want to exclude a space in the second `*`, i.e., `!(* *)`. But then, maybe we want to introduce `noargopts` with a modification to exclude a space. For example,

```bash
local noargopts='!(-*|*[ aCIJjMNPSVWzbEFRDwfGKsTmnO]*)'
```

Reply:
Thank you!